### ISSUE TITLE
No city window upon click

### ISSUE THREAD

**Issue by Samandamand** [yairm210/Unciv#9689]
**Platform**
Android

**Version**
Github 4.7.5

**Describe the bug**
When attempting to touch a city name to bring up the city window nothing happens. I am able access the city menu via the overview menu.

**To Reproduce**
Load up or start a new game in 4.7.5 found a city and try to open the city window.

**Comment by yairm210** [comment]
Interesting, I'll patch

**Comment by yairm210** [comment]
Leaving open so duplicate issues aren't created

**Comment by yairm210** [comment]
No, it's the reverting of the 'only action if touchable' that solves it
The automated units is just too since the original problem in a different way

**Comment by yairm210** [comment]
The "why" is that for buttons that have keypress actions, the keypress actions are activated even if the button is disabled - so you can think you stopped other usages by disabling the button, but you haven't blocked keypress activation
